rest-api-spec/src/main/resources/rest-api-spec/api/indices.simulate_index_template.json

```diff
@@ -1,7 +1,7 @@
 {
   "indices.simulate_index_template":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-templates.html",
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-simulate-template.html",
```

Comment:
```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-simulate-index.html",
```

Reply:
I don’t think that’s right, this API really simulates the template, not the index?

Reply:
Thanks for asking — I double-checked, and I think my suggestion is correct.

The API is named "Simulate index" (which is confusing IMO), but the path, HTTP method, and descriptions all line up. 

The purpose of the API is to test what index templates would be applied to a specific index name.

Reply:
Thanks for taking the time to explain this, and sorry for the noise!